packages/integration-sdk-cli/src/commands/collect.ts

```diff
@@ -30,6 +31,11 @@ export function collect() {
       const enableSchemaValidation = !options.disableSchemaValidation;
       const config = prepareLocalStepCollection(await loadConfig(), options);
       log.info('\nConfiguration loaded! Running integration...\n');
+
+      const graphObjectStore = new FileSystemGraphObjectStore({
+        prettyFile: true,
```

Comment:
There are multiple files right? I wonder if `prettyFiles` would be better. I don't care too much though, just summoning my inner Phil.

Reply:
I agree with ~~Phil~~ Charlie.

Reply:
`prettifyFiles`?